clj-refactor.el

```diff
@@ -1054,14 +1054,18 @@ word test in it and whether the file lives under the test/ directory."
       (s-ends-with? ".cljc" (buffer-file-name))))
 
 (defun cljr--add-ns-if-blank-clj-file ()
+  (interactive)
   (ignore-errors
     (when (and cljr-add-ns-to-blank-clj-files
                (cljr--clojure-ish-filename-p (buffer-file-name))
                (= (point-min) (point-max)))
-      (clojure-insert-ns-form)
-      (newline 2)
-      (when (cljr--in-tests-p)
-        (cljr--add-test-declarations)))))
+      ;; TODO: replace (if ...) with (cider-expected-ns) once dependable
+      (let* ((ns (if (fboundp #'cider-expected-ns)
```

Comment:
`cider-expected-ns` uses the repl to inspect the classpath right?  Does it fall back to `clojure-expected-ns` if unconnected?  If not we should also check of `cider-connected-p`.